client/lib/Hero.ts

```diff
@@ -490,10 +490,9 @@ export default class Hero extends AwaitedEventTarget<{
 
   public async flowCommand(
     commandFn: () => Promise<void>,
-    exitState?: IDomState | DomState | IDomStateAllFn,
-    options?: IFlowCommandOptions,
+    optionsOrExitState?: IDomStateAllFn | IFlowCommandOptions,
```

Comment:
Shouldn't optionsOrExitState also accept an instance of DomState?

I'm going to go ahead and merge. We can figure this out later, but I wanted to at least put it out there.